projects/ad7616-st/src/examples/examples_src.mk

```diff
@@ -0,0 +1,22 @@
+ifeq (y,$(strip $(IIO_EXAMPLE)))
+CFLAGS += -DIIO_EXAMPLE=1 -DIIO_SUPPORT -DINTC_DEVICE_ID=1
```

Comment:
even if the result is the same,  would move the definition of INTC_DEVICE_ID in parameters.h. also, any reason why 1 was chosen? I think there is only one interrupt vector controller for stm32 and by default in the other projects it has id 0.

Reply:
> any reason why 1 was chosen

I think it's a typo on my side. I never noticed it since it works like that. Using 0 also work